client/src/components/Donation/donate-form.tsx

```diff
@@ -332,14 +332,25 @@ class DonateForm extends Component<DonateFormProps, DonateFormComponentState> {
     )}:`;
     const showMinimalPayments = isSignedIn && (isMinimalForm || !isDonating);
 
+    const isAsVariant = !isAVariant;
+
     return (
       <>
         <b className={isMinimalForm ? 'donation-label-modal' : ''}>
-          {isAVariant === false && <SecurityLockIcon />}
           {this.getDonationButtonLabel()}:
         </b>
         <Spacer />
-        <div className={'donate-btn-group'}>
+        <fieldset
+          className={`donate-btn-group ${
+            isAsVariant === false ? 'test-btn-group' : ''
+          }`}
+        >
+          {isAsVariant === false && (
```

Comment:
```suggestion
          {isAVariant && (
```

Reply:
this one can stay? since removing the === false would result in displaying the legend for A variant

Reply:
If you want to hide it for the A variant, why not just use `!isAVariant`?

Reply:
it could be. Initially I used === false to avoid showing the wrong variant if isAVariant returns a falsely value.